test/integration/connect/envoy/case-centralconf/verify.bats

```diff
@@ -54,3 +54,23 @@ load helpers
     must_match_in_prometheus_response localhost:1234 \
     '[\{,]envoy_http_conn_manager_prefix="upstream_s2_http"[,}]'
 }
+
+function get_envoy_cluster_config {
```

Comment:
any reason why you left this local instead of adding it to `helpers.bash`?

Reply:
Initially it was because I didn't want to share something that I thought would not be re-used, but I think I ended up making it generic enough that it might get some re-use.

I suspect that putting all the helpers into `helpers.bash` is quickly going to make it impossible to find things in that file, due to the size. I won't worry about that problem for now.